### PR TITLE
[Admin] [HOME] Connect BackEnd Edit

### PR DESCRIPTION
**Pull request by FebryFernando21**
Semua Edit kluar modal
Udah bisa ganti nama Project, Role, User Name
tampilin username diganti jadi name aja
edit token di admin dari userToken jadi adminToken

### CODE REVIEW

File: src/app/components/modal/modal-data-admin-edit/modal-data-admin-edit.component.ts [hunk 1]
```diff
@@ -1,14 +1,70 @@
...
+        'name': this.namaUser,
+      },
+      options
+    )
+      .subscribe(
+        data => {
+          this.theData = data;
+          console.log(this.theData);
+          if (this.theData.success === true) {
+            // window.location.reload();
```
Comment (antoniusfilian) [inline src/app/components/modal/modal-data-admin-edit/modal-data-admin-edit.component.ts]: why this part commented? should it be true if theData.success === true then the page should be refreshed?

File: src/app/pages/admin/main-dashboard/main-dashboard.component.ts [hunk 2]
```diff
@@ -113,17 +113,18 @@ export class MainDashboardComponent implements OnInit {
...
-    const token = localStorage.getItem("userToken");
+    const token = localStorage.getItem("adminToken");
     const headers = new HttpHeaders().set("authorization", token);
     this.httpClient
       .get("https://nameless-cove-75161.herokuapp.com/api/page/admin/home", {
         headers
       })
       .subscribe(
         (response: ResponseObject) => {
+          console.log(response);
```
Comment (antoniusfilian) [inline src/app/pages/admin/main-dashboard/main-dashboard.component.ts]: please delete this console.log

File: src/app/components/modal/modal-data-admin-edit/modal-data-admin-edit.component.ts (hunk 1 above)
Comment (FebryFernando21) [reply src/app/components/modal/modal-data-admin-edit/modal-data-admin-edit.component.ts]: Elu yan comment tadi setan

Comment (antoniusfilian) [reply src/app/components/modal/modal-data-admin-edit/modal-data-admin-edit.component.ts]: i dont understand. speak english please! pardon me

**Pull request merged**